compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/coneDiagnosticToFirDiagnostic.kt

```diff
@@ -175,6 +175,11 @@ private fun ConeSimpleDiagnostic.getFactory(): FirDiagnosticFactory0<FirSourceEl
         DiagnosticKind.IllegalProjectionUsage -> FirErrors.ILLEGAL_PROJECTION_USAGE
         DiagnosticKind.MissingStdlibClass -> FirErrors.MISSING_STDLIB_CLASS
         DiagnosticKind.Other -> FirErrors.OTHER_ERROR
+        DiagnosticKind.IncorrectCharacterLiteral -> FirErrors.INCORRECT_CHARACTER_LITERAL
+        DiagnosticKind.EmptyCharacterLiteral -> FirErrors.EMPTY_CHARACTER_LITERAL
+        DiagnosticKind.TooManyCharactersInCharacterLiteral -> FirErrors.TOO_MANY_CHARACTERS_IN_CHARACTER_LITERAL
+        DiagnosticKind.IllegalEscape -> FirErrors.ILLEGAL_ESCAPE
+        DiagnosticKind.IllegalEscapeSequence -> FirErrors.ILLEGAL_ESCAPE_SEQUENCE
```

Comment:
As I can see ILLEGAL_ESCAPE_SEQUENCE has no usages in FE 1.0 and can be dropped.